sycl/doc/extensions/proposed/sycl_ext_oneapi_annotated_ptr.asciidoc

```diff
@@ -303,11 +309,24 @@ annotated_ptr(T *Ptr, const properties<PropertyListT> &P = properties{}) noexcep
 ----
 |
 Constructs an `annotated_ptr` object. Does not allocate new storage. The
-underlying pointer is initialized with `Ptr`. `P` is used to specify the
-`PropertyListT` type on the class.
+underlying pointer is initialized with `Ptr`.
+
+The new property set `PropertyListT` contains all properties in `P`.
+If there were duplicate property in `P`, the value of this property must be same.
+
+// --- ROW BREAK ---
```

Comment:
I didn't check for this one but for both specs, please make sure that the order of the new ctors in this table is the same as the order in which they are listed in the class skeleton above. This was the case for the existing member functions.

Reply:
done